test/bench/portfft/launch_bench.hpp

```diff
@@ -22,6 +22,7 @@
 #define PORTFFT_BENCH_LAUNCH_BENCH_HPP
 
 #include <cassert>
+#include <limits>
```

Comment:
Where is this needed?

Reply:
I was initially thinking of reporting average as well as the best bandwidth / flops / time.
So initialized the best time to `std::numeric_limits<double>::max`, and calculate the minimum of the times encountered from there.  But then I had second thoughts regarding do we really need it (the aforementioned switcheroo) and removed it, and this is the artefact of the same.

Removed in the latest commit